.github/workflows/deoloy-to-netlify.yml

```diff
@@ -0,0 +1,40 @@
+name: Build and Deploy to Netlify
```

Comment:
typo filename: `s/deoloy-to-netlify.yml/deploy-to-netlify.yml/`

Reply:
oops